graphene/utils/tests/test_subclass_with_meta.py

```diff
@@ -0,0 +1,63 @@
+from ..subclass_with_meta import SubclassWithMeta
+
+
+def test_init_subclass_calls_super():
+    class DefinesInitSubclass:
+        subclass_init_count = 0
+
+        def __init_subclass__(cls, **kwargs):
+            DefinesInitSubclass.subclass_init_count += 1
+            super().__init_subclass__(**kwargs)
+
+    class InheritsAsFirst(SubclassWithMeta, DefinesInitSubclass):
+        pass
+
+    class InheritsAsLast(DefinesInitSubclass, SubclassWithMeta):
+        pass
+
+    assert DefinesInitSubclass.subclass_init_count == 2
```

Comment:
This would fail with`assert 1 == 2` without the added `super()` call here:

https://github.com/graphql-python/graphene/blob/9d96fc4d86d4114cc9bdac301d4c7bc58ac4f5a7/graphene/utils/subclass_with_meta.py#L52